### PR TITLE
fix(a380x): Model update from Repsol Feb 27th, 2024: Cockpit changes

### PR DESCRIPTION
**Pull request by flogross89**
## Summary of Changes
Changes around autobrake and RTO knob/selector, athr. disconnect button, and ISIS now has two separate textures. 
All credits to Repsol and Mike. 

## Screenshots (if necessary)
none

### CODE REVIEW

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/A380_EXTERIOR.xml [hunk 1]
```diff
@@ -35,27 +35,21 @@
             </UseTemplate>
 
             <!-- LH Body Gear -->
-            <UseTemplate Name="ASOBO_GT_Anim">
+            <UseTemplate Name="ASOBO_GEAR_Back_Template">
```
Comment (Benjozork) [inline fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/A380_EXTERIOR.xml:38]: This doesn't look right. Aren't we supposed to use our own LVars?

cc @crocket63

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/A380_EXTERIOR.xml [hunk 2]
```diff
@@ -115,7 +114,7 @@
                 <ANIM_LENGTH>360</ANIM_LENGTH>
             </UseTemplate>
 
-            <!-- Body Gear Rotation-->
+            <!-- Body Gear Roation-->
```
Comment (Benjozork) [inline fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/A380_EXTERIOR.xml:117]: Revert

**Review by Benjozork — CHANGES_REQUESTED** [review]

File: fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/A380_EXTERIOR.xml (hunk 1 above)
Comment (flogross89) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/A380_EXTERIOR.xml:38]: Oh damnit, I'm very sorry, I thought we were working off the latest master here. Will revert these changes, and only leave the door clickspot in.

Comment (flogross89) [reply fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/A380_EXTERIOR.xml]: @crocket63 Is everything in order now?

**Review by tracernz — APPROVED** [review]

**Pull request merged**